src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,24 @@
+import Popup from "./Popup.js";
+export default class PopupWithConfirm extends Popup {
+  constructor({ popupSelector }) {
+    super({ popupSelector });
+  }
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.